opencti-platform/opencti-front/src/private/components/analyses/groupings/GroupingKnowledgeGraph.jsx

```diff
@@ -1033,14 +1092,33 @@ class GroupingKnowledgeGraphComponent extends Component {
       });
     }, relationshipsToRemove);
     R.forEach((n) => {
-      commitMutation({
-        mutation: groupingKnowledgeGraphMutationRelationDeleteMutation,
-        variables: {
-          id: this.props.grouping.id,
-          toId: n.id,
-          relationship_type: 'object',
-        },
-      });
+      fetchQuery(groupingKnowledgeGraphCheckObjectQuery, {
+        id: n.id,
+      })
+        .toPromise()
+        .then(async (data) => {
+          if (
+            deleteObject
+                && !data.stixObjectOrStixRelationship.is_inferred
+                && data.stixObjectOrStixRelationship.groupings.edges.length === 1
```

Comment:
I see we still have some badly configured linter.. .4 spaces instead of 2 here.

Reply:
fixed